Source/OIDLogoutResponse.h

```diff
@@ -0,0 +1,62 @@
+/*! @file OIDLogoutResponse.h
+    @brief AppAuth iOS SDK
+    @copyright
+        Copyright 2015 Google Inc. All Rights Reserved.
+    @copydetails
+        Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+ 
+        http://www.apache.org/licenses/LICENSE-2.0
+ 
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+@class OIDLogoutRequest;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/*! @brief Represents the response to a logout request.
+    @see http://openid.net/specs/openid-connect-frontchannel-1_0.html
+ */
+
+@interface OIDLogoutResponse : NSObject <NSCopying, NSSecureCoding> {
+    OIDLogoutRequest *_request;
+    NSString *_state;
+}
+
+/*! @brief The request which was serviced.
+ */
+@property(nonatomic, readonly) OIDLogoutRequest *request;
+
+/*! @brief REQUIRED if the "state" parameter was present in the client authorization request. The
+        exact value received from the client.
+    @remarks state
+ */
+@property(nonatomic, readonly, nullable) NSString *state;
+
+/*! @internal
+    @brief Unavailable. Please use initWithParameters:.
+ */
+- (instancetype)init NS_UNAVAILABLE;
+
+/*! @brief Designated initializer.
+    @param request The serviced request.
+    @param parameters The decoded parameters returned from the Server.
```

Comment:
what endpoint is this? rather than "from the Server" we should say "from the XYZ endpoint".

Reply:
Good point, will rewrite soon!